app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, first_name, second_name = command.split()
```

Comment:
what if command is wrong and doesn't have exactly three elements?

Reply:
you need to check if split-command has exactly three elements otherwise your program will crash during unpacking

Reply:
I'd suggest to apply `try` block only to file-related operations - starting from when you open file.